rust/src/shards/gui/mod.rs

```diff
@@ -31,6 +31,7 @@ static FLOAT_VAR_OR_NONE_SLICE: &[Type] = &[
   common_type::none,
 ];
 static FLOAT2_VAR_SLICE: &[Type] = &[common_type::float2, common_type::float2_var];
+static HASH_VAR_OR_NONE_SLICE: &[Type] = &[common_type::none, common_type::int2_var];
```

Comment:
Shouldn't it be named `INT2_VAR_OR_NONE_SLICE` or do you want to have the actual type an implementation details so that we use the same type everywhere we need a hash, but it could become something else than int2?

Reply:
Yeah that was the idea, if it was INT2 would better put to types.rs also, (in fact we should unify at some point)